### PR TITLE
add emoji, and mention autocomplete support

### PR DESCRIPTION
**Pull request by vevix**
Emoji
--------

Users
--------

We can extend this to media later on :open_mouth:

### CODE REVIEW

File: app/frontend/app/utils/suggester.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        return `$1:${value}: `;
+      }
+    },
+    { // mentions
+      cache: true,
+      debounce: 500,
+      match: /(^|\s)@(\w*)$/,
+      search: function(term, callback) {
+        // show some staff when user enters just '@'
+        let names = ['Josh', 'Nuck', 'ryn', 'Cybrox', 'vevix'];
```
Comment (cybrox) [inline app/frontend/app/utils/suggester.js]: You capitalized my name! :skull: :stuck_out_tongue:

**Comment by vevix** [comment]
Hmm odd, I swear it was capitalized on HB.

File: app/frontend/app/utils/suggester.js (hunk 1 above)
Comment (vevix) [inline app/frontend/app/utils/suggester.js]: Hmm odd, I swear it was capitalized on HB.

**Comment by JoshFabian** [comment]
Holy crap, so excited for this. Cross-browser support or #chromemasterrace only?

**Comment by Wopian** [comment]
Wopian masterrace

**Comment by NuckChorris** [comment]
lgtm

@josh should be all, it's just Twemoji

**Comment by JoshFabian** [comment]
Twemoji beats Hangouts? What sort of world are we living in?

**Comment by vevix** [comment]
A blobless world

**Comment by vevix** [comment]
@Josh yep, Anime/Manga or anything on our search TBH.

Using my fork of Twemoji too so we can support custom emoji.

**Comment by NuckChorris** [comment]
Add :josh:

**Comment by JoshFabian** [comment]
Oh man, that would be great!

**Pull request merged**